pkg/cassandra/gocql/testutils/udt_test.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
you can remove this and run `make fmt`

Reply:
![Screenshot 2024-01-30 114037](https://github.com/jaegertracing/jaeger/assets/103821431/18c93f7e-7a8b-42de-b212-48f48f821eed)
Not able to run make fmt. Executed make install-tools, and it did install gofumpt from mydvan but still saying command not found.

Reply:
i think you'll need to install `gofumpt`, you can install gofumpt using this command `go install mvdan.cc/gofumpt@latest`

Reply:
Yes, I did install that using the exact same command

Reply:
run `make fmt`